documentation/certificates.md

```diff
@@ -30,6 +50,9 @@ OpenConnect CA:
    \__ OpenConnect client 1
    \__ OpenConnect client ...
    \__ OpenConnect client n
+      \__ OpenConnect client PKCS 1
```

Comment:
I would omit the PKCS#11 stuff from this hierarchy - probably too complicated to fit in nicely. It's the same client certs in a diff format. The way you have it here also makes it seem like the PKCS#11 format clients certs are under the "Openconnect client n" hierarchy when instead they're adjacent leafs to each of the "client" entries.